cypress/integration/portal-launch.test.ts

```diff
@@ -73,6 +73,53 @@ context("Launch AP From the Portal", () => {
     cy.get("[data-cy=activity-summary]").should("contain", "Single Page Test Activity");
   });
 
+  it("can be launched collaboratively", () => {
```

Comment:
The before each is also launching an activity. It guess it would be better if the before each just logged in the user.
And then the two non-collaborative lauches are put in their own context block with a before each that launches the activity. 

And this collaborative launch lives outside, or you could give it, its own context block.